File: ProgramacionUR/PARCIAL1/Punto2.cpp
#include<iostream>
#include<math.h>
#include<iomanip>

using namespace std;

int main()
{
    cout<<fixed<<setprecision(14);
    float numerador_total, num_raiz, denominador_total, denom_num, denom_denom, z;
    float x=0.0;
    float y=0.0;
    
    cout<<"Ingrese x:"<<endl;
    cin>>x;
    cout<<"Ingrese y:"<<endl;
    cin>>y;
    
    num_raiz=sqrt(pow(y,x)+3)/8;
    numerador_total=sqrt(pow(x,y)*num_raiz);
    
    denom_num=12*y-2*pow(x,2+y);
    denom_denom=(32*y-5)/pow(x*y,2);
    denominador_total=pow(denom_num/denom_denom,3);
    z=pow(numerador_total/denominador_total,3);
    cout<<"z: "<<z<<endl;
    return 0;
}

File: ProgramacionUR/PARCIAL1/Punto2.py
import math

numerador_total=0
num_raiz=0
denominador_total=0
denom_num=0
denom_denom=0
z=0
x=0.0
y=0.0
    
x=float(input("Ingrese x:"))
y=float(input("Ingrese y:"))
    
num_raiz=math.sqrt((y**x)+3)/8
numerador_total=math.sqrt((x**y)*num_raiz);
    
denom_num=12*y-2*(x**(2+y))
denom_denom=(32*y-5)/((x*y)**2);
denominador_total=(denom_num/denom_denom)**3;
z=(numerador_total/denominador_total)**3;
print("z: ",z)